markdown_lab/network/client.py

```
@@ -201,6 +206,143 @@ def _request_with_retries(
+                            f"attempt: {attempt + 1})"
+                        )
+
+                        return content
+
+                except Exception as e:
+                    last_exception = e
+
+                    if attempt < self.config.max_retries:
+                        wait_time = 2**attempt  # Exponential backoff
+                        logger.warning(
+                            f"Async request failed for {url} on attempt {attempt + 1}/{self.config.max_retries + 1}: "
```

Pylint (3.3.7)

[convention] Line too long (118/100)

(C0301)